app/models/monthly_report.rb

```diff
@@ -76,18 +71,11 @@ def set_prev_monthly_report!
 
   private
 
-  def registrable_term_from
-    REGISTRABLE_TERM_FROM
-  end
-
-  def registrable_term_to
-    Time.current.last_month.since(5.days)
-  end
-
   def target_month_registrable_term
+    return if user.blank?
     return if target_month.blank?
     return if registrable_term?
-    errors.add :target_month, " must be #{registrable_term}"
+    errors.add :target_month, " must included by user.report_registrable_months"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)